src/cython/vapoursynth.pyx

```diff
@@ -1956,9 +1956,15 @@ cdef class RawNode(object):
     def node_name(self):
         return self.funcs.getNodeName(self.node).decode("utf-8")
 
-    @property
-    def timings(self):
-        return self.funcs.getNodeProcessingTime(self.node, 0)
+    property timings:
+        def __get__(self):
+            return self.funcs.getNodeProcessingTime(self.node, False)
+        
+        def __set__(self, int value):
```

Comment:
why not "bint reset" if all you can do is pass 0 anyway?

Reply:
No particular reason other than int is used in VSAPI too, I'll change it.